packages/discord.js/typings/index.d.ts

```diff
@@ -6542,6 +6558,23 @@ export interface PartialMessage
 
 export interface PartialMessageReaction extends Partialize<MessageReaction, 'count'> {}
 
+export interface PartialPoll
+  extends Partialize<
+    Poll,
+    'allowMultiselect' | 'layoutType' | 'expiresTimestamp',
+    null,
+    'question' | 'message' | 'answers'
+  > {
+  question: { text: null };
+  message: PartialMessage;
```

Comment:
Can't message also be.. a full message?

Reply:
If you had a full message, you'd also have a full poll, as the poll is part of the message (goes both ways)